tests/visco_plastic/screen-ouput

```diff
@@ -0,0 +1,66 @@
+-----------------------------------------------------------------------------
+-- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
+--     . version 1.4.0-pre
+--     . running in DEBUG mode
+--     . running with 1 MPI process
+--     . using Trilinos
+-----------------------------------------------------------------------------
+
+Number of active cells: 10,000 (on 1 levels)
+Number of degrees of freedom: 171,805 (80,802+10,201+40,401+40,401)
```

Comment:
do you need a problem with that many unknowns? The cheaper the tests are the better.